app/design/frontend/base/default/template/ebanx/form/baloto.phtml

```diff
@@ -1,13 +1,16 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-baloto.png" width="40" /></div>';
```

Comment:
I think we can improve it and put something like:

```php
<?php if ($showInlineIcon): ?>
    <div><img src="<?= $skin ?>frontend/base/default/ebanx/ebanx-baloto.png" width="40" /></div>
<?php endif ?>
```

This way, we can remove some unnecessary variables, IMO.